packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialogWrapper.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
it's minor but this is the official header now.

`/*
 * Tupaia
 * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
 */`